app/src/main/java/org/thoughtcrime/securesms/trustedIntroductions/glue/AttachmentTableGlue.kt

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2024 Signal Messenger, LLC
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+package org.thoughtcrime.securesms.trustedIntroductions.glue
+
+import androidx.annotation.NonNull
+import org.thoughtcrime.securesms.attachments.Attachment
+import org.thoughtcrime.securesms.trustedIntroductions.TI_Utils
+import java.io.ByteArrayOutputStream
+import java.io.InputStream
+
+object AttachmentTableGlue {
+
+
+  /**
+   * TODO
+   *
+   * @param attachment the attachment to be evaluated
+   * @return a ByteinputStream containing the contents of the attachment
+   */
+  @JvmStatic
+  fun grabIntroductionData(attachment: Attachment, inputStream: InputStream): InputStream {
+    var text = ""
+    if(attachment.fileName!!.contains(TI_Utils.TI_MESSAGE_EXTENSION)){
+      val byteOutputStream = ByteArrayOutputStream()
```

Comment:
I think it is good practice to have comments or logs to show what is going on

Reply:
Finished the doc string which was still marked as TODO with some explanations.